src/platform/i386/chal_pgtbl.c

```diff
@@ -0,0 +1,188 @@
+#include "shared/cos_types.h"
+#include "captbl.h"
+#include "pgtbl.h"
+#include "cap_ops.h"
+#include "liveness_tbl.h"
+#include "retype_tbl.h"
+
```

Comment:
Why are the `cons` and `decons` functions not in the chal?

Reply:
These will be moved to chal in the commits that follows.